src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -113,7 +113,12 @@ const Body = BaseFormWithPolling.extend(
               return this.trigger('save', this.model);
             }).fail((xhr) => {
               Logger.error(`OV challenge response with HTTP code ${xhr.status} ${xhr.responseText}`);
-              onPortFail();
+              if (xhr.statusText === 'timeout') {
```

Comment:
create a constant
CHALLENGE_TIMEOUT = 'timeout'

Reply:
Hey @pradeepdewda-okta I feel we might not want to make a const for this since there is only one reference of 'timeout' in this file. Ideally, we might want to have a constants collection object somewhere at framework/util level to capture all keyword in JavaScript. for example,
'object', 'image', 'function', 'title' etc. today, I saw we use these string (not const) everywhere.
but it is out of the scope of this issue.